php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
I think it would be greatly beneficial if there was not a single "MONGOC_EXPERIMENTAL_FEATURES" flag. I can imagine, this specific flag meaning different things with different versions of mongo-c-driver (in the future). As this is somewhat of a feature flag, there should be one for each specific thing that we can turn on and off. In this case, I would suggest to have something akin to MONGOC_HAVE_METADATA. 

It also seems a bit strange to have a patch that introduces a feature, but *also* disables that same feature through a config.m4 flag.